docs/upgrading-eck.asciidoc

```diff
@@ -7,131 +7,23 @@ endif::[]
 == Upgrading ECK
 
 [float]
-[id="{p}-upgrade-to-v1beta1"]
-=== Upgrading to ECK 1.0.0-beta1 from previous versions
+[id="{p}-ga-upgrade"]
+=== Upgrading to ECK 1.0.0 from previous versions
 
-ECK 1.0.0-beta1 includes changes that are incompatible with previous versions of the operator. Notable changes include:
+ECK 1.0.0 is largely compatible with the beta version of the operator (see <<{p}-ga-openshift,the exceptions for earlier Kubernetes and OpenShift versions>>). There is not a direct upgrade path from the alpha version.
 
-- Custom resource version has changed from `v1alpha1` to `v1beta1`.
-- Some CRD fields have been removed and some others have been renamed to clarify their purpose.
-- Elasticsearch cluster orchestration is now managed through link:https://kubernetes.io/docs/concepts/workloads/controllers/statefulset/[StatefulSet] resources.
+To upgrade from 1.0.0-beta1, follow the <<{p}-quickstart>>. Note that all Elasticsearch pods will go through a rolling restart when the operator is upgraded. Once upgraded, you should use the `v1` API version for all Elastic manifests.
```

Comment:
There are two ideas on the same line:
* all Elasticsearch Pods go through a rolling restart when the operator is upgraded
* To upgrade from 1.0.0-beta1, follow the <<{p}-quickstart>>

I would put them on 2 separate lines to avoid confusion.